search/models.go

```diff
@@ -51,6 +51,6 @@ type SearchRequestOptions struct {
 	// We use this data for a further filtering out of hits recieved from 'grammar filter' search that duplicates the carousel items.
 	// Since the search for 'grammar filter' is async. to  classification intents (carousel) search, we don't have yet the data for filterOutCUSources fild.
 	includeTypedUidsFromContentUnits bool
-	// If not nil, set how long a search is allowed to take, e.g. "1s" or "500ms".
+	// If not nil, set how long a search is allowed to take, e.g. "1s" or "500ms". Note: Not always respected by ES.
```

Comment:
This is why you did this PR? Because ES not always respect the timeout?

Reply:
yes